src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public abstract void draw();
+
+    public abstract double getArea();
```

Comment:
and you don't need these abstract methods in this class, only subclasses must implement them)